### ISSUE TITLE
is it possible to add simple categories to these companies

### ISSUE THREAD

**Issue by YuFengDing** [remoteintech/remote-jobs#1598]
With growth of the list, it's difficult to pick up the suitable company quickly by just presenting the name and official website of the company in order.

is it possible to add simple categories,e.g.  industries,scale, financing status, established time
[Discussions > New > Ideas](https://github.com/remoteintech/remote-jobs/discussions/new?category=ideas)

**Comment by zessu** [comment]
Yes I think this is a great idea. We can add information like industry for example

**Comment by souad988** [comment]
@dougaitken  would you please assign this issue to me

**Comment by dougaitken** [comment]
@souad988 before assigning any Issues, I would like possible contributors to discuss their solution in the Idea for enhancements

**Comment by dougaitken** [comment]
Closing this older issue - I've recently made some big changes to the site and many things are now filterable or are categories
